src/settings/FixedDueDateSchedule/FixedDueDateScheduleForm.test.js

```diff
@@ -0,0 +1,512 @@
+  afterEach(() => {
+    jest.clearAllMocks();
+  });
+
+  describe('Initial render with default props', () => {
+    let container;
+    const fieldCallOrder = {
+      name: 1,
+      description: 2,
+    };
+    const accordionCallOrder = {
+      general: 1,
+      schedule: 2,
+    };
+    const expandAllButtonCallOrder = {
+      render: 1,
+      update: 2,
+    };
+    const buttonCallOrder = {
+      cancel: 1,
+      save: 2,
+    };
+    const accordionSetCallOrder = {
+      render: 1,
+      update: 2,
+    };
+
+    beforeEach(() => {
+      container = render(
+        <FixedDueDateScheduleForm {...defaultProps} />
+      );
+    });
+
+    it('"IconButton" should be rendered with correct props', () => {
+      const expectedProps = {
+        'aria-label': [messageIds.closeLabel],
+        icon: 'times',
+        iconClassName: 'closeIcon',
+        onClick: defaultProps.onCancel,
+      };
+
+      expect(IconButton).toHaveBeenCalledWith(expect.objectContaining(expectedProps), {});
+    });
+
+    it('"onCancel" should be called after clicking on close icon', () => {
+      fireEvent.click(container.getByTestId('close-icon'));
+
+      expect(defaultProps.onCancel).toBeCalled();
+    });
+
+    it('"Pane" component should have correct title', () => {
+      expect(container.getByText(messageIds.newFixDDSchedule)).toBeInTheDocument();
+    });
+
+    it('"Field" for name should be called with correct props', () => {
+      const expectedProps = {
+        id: 'input_schedule_name',
+        component: TextField,
+        name: 'name',
+        required: true,
+        autoFocus: true,
+      };
+
+      expect(Field).toHaveBeenNthCalledWith(fieldCallOrder.name, expect.objectContaining(expectedProps), {});
+      expect(container.getByText(messageIds.name)).toBeInTheDocument();
+    });
+
+    it('"Field" for description should be called with correct props', () => {
+      const expectedProps = {
+        component: TextArea,
+        name: 'description',
+        fullWidth: true,
+      };
+
+      expect(Field).toHaveBeenNthCalledWith(fieldCallOrder.description, expect.objectContaining(expectedProps), {});
+      expect(container.getByText(messageIds.description)).toBeInTheDocument();
+    });
+
+    it('"FieldArray" should be called with correct props', () => {
+      const expectedProps = {
+        timezone: stripes.timezone,
+        component: SchedulesList,
+        name: 'schedules',
+      };
+
+      expect(FieldArray).toHaveBeenCalledWith(expect.objectContaining(expectedProps), {});
+    });
+
+    it('"General" accordion should be rendered with correct props', () => {
+      const expectedProps = {
+        open: defaultState.sections.generalFixedDueDate,
+        id: 'generalFixedDueDate',
+      };
+      expect(Accordion)
+        .toHaveBeenNthCalledWith(accordionCallOrder.general, expect.objectContaining(expectedProps), {});
+      expect(container.getByText(messageIds.about)).toBeInTheDocument();
```

Comment:
same here. please check all other places, I believe all such places must be separate tests